src/pages/designing/kits/figma.mdx

```diff
@@ -55,40 +58,58 @@ Then in the Libraries modal that appears, find the team called
 </Column>
 </Row>
 
-To preview the four themed libraries, visit these view-only files.
+<br />
+<br />
+
+To preview the four Carbon themes, visit these view-only links.
 
 <Row className="resource-card-group">
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('P0OEN9TS', 0)}
       subTitle="White theme"
       href="https://www.figma.com/file/Vzz8k68Pqk5HfaTdQOQrGu/White-Theme---Carbon-Design-System">
+      actionIcon="launch">
+
+![Figma icon](images/figma.svg)
+
     </ResourceCard>
+
   </Column>
   <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      onClick={() => fathom.trackGoal('T7D1HJ3L', 0)}
       subTitle="Gray 10 theme"
       href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System">
+      actionIcon="launch">
```

Comment:
Looks like the problem is that you forgot to delete the old closing tag when adding the new line. You can add back the onClick.
```suggestion
      href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System"
      actionIcon="launch">
```

Reply:
Oh thank you @jnm2377 ! Hopefully this fixes it!